core/actions/_lib/validateFields.ts

```diff
@@ -28,8 +30,21 @@ export const validatePubValuesBySchemaName = ({
 			errors[slug] = `Field ${field.slug} does not have a schemaName, cannot validate.`;
 			continue;
 		}
-		const jsonSchema = getJsonSchemaByCoreSchemaType(field.schemaName);
-		const result = Value.Check(jsonSchema, value);
+
+		let result = false;
+		// Rich text fields are a special case where we use prosemirror to validate
+		// as opposed to typebox
+		if (field.schemaName === CoreSchemaType.RichText) {
+			try {
+				(value as Node).check();
```

Comment:
I really don't know if this will work yet...

Reply:
I think, as the `value` will be a `JSON` value, we will first need to call `nodeFromJSON` on the `schema` @isTravis defines here: https://github.com/pubpub/platform/blob/tr/context-editor/packages/context-editor/src/schemas/index.ts

Not sure how you would do this in this PR though 😅. Maybe you extract the prosemirror `nodeFromJSON` + `Node.check` validation to a different file/function, and create a stub schema there for now with a comment indicating we will import it from the editor?